client/packages/invoices/src/Returns/DetailView/SidePanel/AdditionalInfoSection.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC, memo } from 'react';
+import {
+  Grid,
+  DetailPanelSection,
+  PanelField,
+  PanelLabel,
+  PanelRow,
+  //   BufferedTextArea,
+  useTranslation,
+  //   ColorSelectButton,
+  //   useBufferState,
+  InfoTooltipIcon,
+} from '@openmsupply-client/common';
+import { useReturns } from '../../api';
+
+export const AdditionalInfoSectionComponent: FC = () => {
+  const t = useTranslation('replenishment');
+  const { data } = useReturns.document.outboundReturn();
+  //   const isDisabled = useOutbound.utils.isDisabled();
+  //   const { colour, comment, user, update } = useOutbound.document.fields([
+  //     'colour',
+  //     'comment',
+  //     'user',
+  //   ]);
+  //   const [colorBuffer, setColorBuffer] = useBufferState(colour);
+  //   const [commentBuffer, setCommentBuffer] = useBufferState(comment ?? '');
+
+  //  TO-DO: Make this work
+  const { user } = data || {};
```

Comment:
I copied this from your one, but there's no `user` node on either of our ReturnByNumber queries (that I could tell). So not sure how you got the user data to be a valid type. Can probably figure out before merging, just wanted to rush this out the door for now.